src/channelsyncroniser.ts

```diff
@@ -169,15 +169,23 @@ export class ChannelSyncroniser {
         try {
             rooms = await this.GetRoomIdsFromChannel(channel);
         } catch (err) { } // do nothing, our rooms array will just be empty
+        let fallbackAlias = "";
         for (const room of rooms) {
             try {
                 const al = (await this.bridge.getIntent().getClient()
                     .getStateEvent(room, "m.room.canonical_alias")).alias;
                 if (al) {
-                    return al; // we are done, we found an alias
+                    if (al.startsWith("#_discord_")) {
```

Comment:
This feels horribly hacky

Reply:
We do similar things all over already. Once we switch to the matrix-js-bot-sdk we can use `isNamespacedAlias` or whatever that was called instead